### PR TITLE
Task-144055 : build script updated for ewma package.

### PR DESCRIPTION
**Pull request by vikasgupta8**
Updated script for package ewma.

### CODE REVIEW

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh [hunk 1]
```diff
@@ -15,110 +15,54 @@
...
-	if ! go test ./...; then
-		test_with_master
-		exit 0
-	else
-		echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
-		echo "$PACKAGE_VERSION $PACKAGE_NAME" > /home/tester/output/test_success 
-		echo "$PACKAGE_NAME  |  $PACKAGE_VERSION | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" > /home/tester/output/version_tracker
-		exit 0
+if [ "$PACKAGE_VERSION" = "v1.2.0" ]; then
+	echo "VIKAS version is v1.2.0"
```
Comment (amitsadaphule) [inline g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh:38]: Please cleanup the logs.

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh [hunk 2]
```diff
@@ -15,110 +15,54 @@
...
+	git clone --recurse https://github.com/vividcortex/ewma
+	git checkout $PACKAGE_VERSION
+	cd $PACKAGE_NAME
```
Comment (amitsadaphule) [inline g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh:41]: The command sequence is incorrect. Should have been:

```
cd $PACKAGE_NAME
git checkout $PACKAGE_VERSION
```

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh [hunk 3]
```diff
@@ -15,110 +15,54 @@
...
+
 echo "Testing $PACKAGE_PATH with $PACKAGE_VERSION"
+
 # Ensure go.mod file exists
-go mod init
-if ! go test ./...; then
-	test_with_master
-	exit 0
+go mod init github.com/vividcortex/ewma
+go mod tidy
```
Comment (amitsadaphule) [inline g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh:57]: This seems unnecessary for `v1.2.0` which is cloned and already has `go.mod`.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh (hunk 1 above)
Comment (vikasgupta8) [reply g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh]: Done

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh (hunk 3 above)
Comment (vikasgupta8) [reply g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh]: Corrected the code for v1.2.0

File: g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh (hunk 2 above)
Comment (vikasgupta8) [reply g/github.com__vividcortex__ewma/github.com__vividcortex__ewma_rhel_8.3.sh:39]: Corrected the sequence.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Comment by amitsadaphule** [comment]
Please do not specify RTC task number in commits.

**Pull request merged**